data/FilterList.json

```diff
@@ -1759,8 +1759,8 @@
   {
     "id": 146,
     "description": "Adblock Plus for Japan domestic and Japanese correspondence Universal Filter.",
-    "descriptionSourceUrl": "http://www.schuzak.jp/other/abp.html",
-    "homeUrl": "http://www.schuzak.jp/other/abp.html",
+    "descriptionSourceUrl": "http://web.archive.org/web/20120419124052/http://www.schuzak.jp/other/abp.html",
+    "homeUrl": "http://web.archive.org/web/20120419124052/http://www.schuzak.jp/other/abp.html",
```

Comment:
If a list is still available on the Internet Archive, we have traditionally kept them just as a reference. I would say let's keep it. Thanks.